machine/v1beta1/types_awsprovider.go

```diff
@@ -257,30 +257,10 @@ type AWSMachineProviderStatus struct {
 	Conditions []AWSMachineProviderCondition `json:"conditions,omitempty" protobuf:"bytes,3,rep,name=conditions"`
 }
 
-// AWSMachineProviderConditionType is a valid value for AWSMachineProviderCondition.Type
-type AWSMachineProviderConditionType string
-
-// Valid conditions for an AWS machine instance.
-const (
-	// MachineCreation indicates whether the machine has been created or not. If not,
-	// it should include a reason and message for the failure.
-	MachineCreation AWSMachineProviderConditionType = "MachineCreation"
-)
-
-// AWSMachineProviderConditionReason is reason for the condition's last transition.
-type AWSMachineProviderConditionReason string
-
-const (
-	// MachineCreationSucceeded indicates machine creation success.
-	MachineCreationSucceeded AWSMachineProviderConditionReason = "MachineCreationSucceeded"
-	// MachineCreationFailed indicates machine creation failure.
-	MachineCreationFailed AWSMachineProviderConditionReason = "MachineCreationFailed"
-)
-
 // AWSMachineProviderCondition is a condition in a AWSMachineProviderStatus.
 type AWSMachineProviderCondition struct {
 	// Type is the type of the condition.
-	Type AWSMachineProviderConditionType `json:"type" protobuf:"bytes,1,opt,name=type,casttype=AWSMachineProviderConditionType"`
+	Type ConditionType `json:"type" protobuf:"bytes,1,opt,name=type,casttype=AWSMachineProviderConditionType"`
```

Comment:
Drop all of the protobuf tags, for CRDs we're not benefiting from them at all. Also this one is outdated due to type name changes.

Reply:
I'd suggest creating a separate commit dropping all of protobuf tags from all files under `machine` directory.

Reply:
I've just noticed that if I run `make generate-with-container` protobuf tags are automatically added to API fields.

Reply:
Problem solved, I removed machine api from protobuf generation config